### DISCUSSION TITLE
这个错误是怎么产生的

### DISCUSSION THREAD

**Discussion by Matiz7** [PaddlePaddle/PaddleOCR#13593] [Show and tell]
在阿里云函数计算环境下，发生频率很低
--------------------------------------                                                                     
C++ Traceback (most recent call last):                                                                     
--------------------------------------                                                                     
No stack trace in paddle, may be caused by external reasons.                                               
                                                                                                           
----------------------                                                                                     
Error Message Summary:                                                                                     
----------------------                                                                                     
FatalError: `Termination signal` is detected by the operating system.                                      
  [TimeInfo: *** Aborted at 1722822501 (unix time) try "date -d @1722822501" if you are using GNU date ***]
  [SignalInfo: *** SIGTERM (@0x0) received by PID 1 (TID 0x7fc03a26f740) from PID 0 ***]

**Comment by GreatV** [comment]
什么paddleocr版本，什么paddle版本呀？

**Reply by Matiz7** [reply]
Python3.10 / paddle2.5.0 / paddleocr 2.7.3.0

**Reply by GreatV** [reply]
升级一下版本试试

**Reply by Matiz7** [reply]
升级paddle的版本 ？之前试过2.6好像还不成熟 ocr的版本已经是最新的了。

**Reply by GreatV** [reply]
可能是cpu指令集不支持nonavx512，建议升级到paddle3.0beta
